File: sunbin/day12/python/001_옷가게할인받기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/120818

def solution(price):
    if price>=500000:
        price = price *0.8
    elif price>=300000:
        price = price *0.9
    elif price>=100000:
        price = price * 0.95
        
    return int(price)


print(solution(350000))

File: sunbin/day12/001_옷가게할인받기.js
// https://school.programmers.co.kr/learn/courses/30/lessons/120818

function solution(price) {
  let answer = price;
  if (price >= 500000) {
    answer = price * 0.8;
  } else if (price >= 300000) {
    answer = price * 0.9;
  } else if (price >= 100000) {
    answer = price * 0.95;
  }

  return parseInt(answer);
}